shaptools/cli.py

```diff
@@ -0,0 +1,350 @@
+    copy_ssfs.add_argument('--remote_host', help='Other host name', required=True)
+    copy_ssfs.add_argument(
+        '--remote_password',
+        help='Other host SAP HANA instance password (sid and instance must match '\
+        'with the current host)', required=True)
+
+
+# pylint:disable=W0212
+def uninstall(hana_instance, logger):
+    """
+    Uninstall SAP HANA database instance
+    """
+    logger.info(
+        'This command will uninstall SAP HANA instance '\
+        'with sid %s and instance number %s (y/n): ',
+        hana_instance.sid, hana_instance.inst)
+    response = input()
+    if response == 'y':
+        user = hana.HanaInstance.HANAUSER.format(sid=hana_instance.sid)
+        hana_instance.uninstall(user, hana_instance._password)
+    else:
+        logger.info('Command execution canceled')
+
+
+def run_hdbsql(hana_instance, hana_args, cmd):
+    """
+    Run hdbsql command
+    """
+    hdbsql_cmd = hana_instance._hdbsql_connect(
+        key_name=hana_args.key_name,
+        user_name=hana_args.user_name,
+        user_password=hana_args.user_password)
+    cmd = '{hdbsql_cmd} {database}\\"{cmd}\\"'.format(
+        hdbsql_cmd=hdbsql_cmd,
+        database='-d {} '.format(hana_args.database) if hana_args.database else '',
+        cmd=cmd)
+    hana_instance._run_hana_command(cmd)
+
+def run_hana_subcommands(hana_instance, hana_args, logger):
+    """
+    Run hana subcommands
+    """
+    str_args = hana_args.hana
+    if str_args == 'is_running':
+        result = hana_instance.is_running()
+        logger.info('SAP HANA database running state: %s', result)
+    elif str_args == 'version':
+        hana_instance.get_version()
+    elif str_args == 'start':
+        hana_instance.start()
+    elif str_args == 'stop':
+        hana_instance.stop()
+    elif str_args == 'info':
+        hana_instance._run_hana_command('HDB info')
+    elif str_args == 'kill':
+        hana_instance._run_hana_command('HDB kill-9')
+    elif str_args == 'overview':
+        hana_instance._run_hana_command('HDBSettings.sh systemOverview.py')
+    elif str_args == 'landscape':
+        hana_instance._run_hana_command('HDBSettings.sh landscapeHostConfiguration.py')
+    elif str_args == 'uninstall':
+        uninstall(hana_instance, logger)
+    elif str_args == 'dummy':
+        run_hdbsql(hana_instance, hana_args, 'SELECT * FROM DUMMY')
+    elif str_args == 'hdbsql':
+        run_hdbsql(hana_instance, hana_args, hana_args.query)
+    elif str_args == 'user':
+        hana_instance.create_user_key(
+            hana_args.key_name, hana_args.environment, hana_args.user_name,
+            hana_args.user_password, hana_args.database)
+    elif str_args == 'backup':
+        hana_instance.create_backup(
+            hana_args.database, hana_args.name, hana_args.key_name,
+            hana_args.user_name, hana_args.user_password)
+
+
+def run_sr_subcommands(hana_instance, sr_args, logger):
```

Comment:
what is `sr`?

Reply:
System Replication?

Reply:
Yes, sr stands for System replication

Reply:
Exactly! Thanks @fmherschel

Reply:
We would just maybe add comment for the acronym

Reply:
@MalloZup There is a `help` output for that (`-h` option).